Fix build failure against gcc-4.7 (missing <unistd.h> for close())

../client/SecureID.cpp: In member function 'std::string SecureID::GetPrivateKey()':
../client/SecureID.cpp:92:13: error: 'close' was not declared in this scope
../client/SecureID.cpp:102:14: error: 'close' was not declared in this scope
../client/SecureID.cpp:115:13: error: 'close' was not declared in this scope
../client/SecureID.cpp:124:12: error: 'close' was not declared in this scope

make[3]: *** [SecureID.o] Error 1
diff --git a/src/client/client/SecureID.cpp b/src/client/client/SecureID.cpp
index bc276c4..5eb2f13 100644
--- a/src/client/client/SecureID.cpp
+++ b/src/client/client/SecureID.cpp
@@ -75,6 +75,7 @@ std::string SecureID::GetPrivateKey(void)
 
 #include <sys/ioctl.h>
 #include <net/if.h>
+#include <unistd.h>
 
 std::string SecureID::GetPrivateKey(void)
 {
